rules/windows/execution_shared_modules_local_sxs_dll.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2020/10/28"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation, change, or deletion of a DLL module within a Windows SxS local folder. Adversaries may abuse shared modules to
+execute malicious payloads by instructing the Windows module loader to load DLLs from arbitrary local paths.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Execution via local SxS Shared Module"
+note = "The SxS DotLocal folder is a legitimate feature that can be abused to hijack standard modules loading order by forcing an executable on the same application.exe.local folder to load a malicious DLL module from the same directory."
+references = ["https://docs.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-redirection"]
+risk_score = 43
+rule_id = "a3ea12f3-0d4e-4667-8b44-4230c63f3c75"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution"]
+type = "eql"
+
+query = '''
+file where wildcard(file.path, "C:\\*\\*.exe.local\\*.dll")
```

Comment:
```suggestion
file where file.path : "C:\\*\\*.exe.local\\*.dll"
```

Reply:
can you add `file.extension : "dll"`, too?